.github/workflows/download_openssl.py

```diff
@@ -12,7 +12,7 @@ def get_response(session, url, token):
     response = session.get(url, headers={"Authorization": "token " + token})
     if response.status_code != 200:
         raise ValueError("Got HTTP {} fetching {}: ".format(
-            response.status_code, url, response.content
```

Comment:
juh?

Reply:
flake8 started complaining about this, it was always wrong.